.coafile

```diff
@@ -44,16 +44,16 @@ body_close_issue_full_url = True
 body_close_issue_on_last_line = True
 body_enforce_issue_reference = False
 
-[LineCounting]
+[all.LineCounting]
 enabled = False
 bears = LineCountBear
 max_lines_per_file = 1000
 
-[rst]
+[all.rst]
```

Comment:
IMO this section can be independent and doesn't need to be inherited from all since you are replacing the files.